src/java/Civil.java

```diff
@@ -0,0 +1,31 @@
+package java;
+
+import java.util.Scanner;
+
+public class Civil extends Air_transport {
+    int NumberOfPassengers;
+    boolean BusinessClass;
+
+    @Override
+    void Vyvod() {
+        super.Vyvod();
+        System.out.println("Number of Passengers: " + NumberOfPassengers);
+        System.out.println("Businees Class: " + BusinessClass);
+    }
+    void loading(){
+        System.out.println("Enter the number of passengers: ");
+        Scanner scanner=new Scanner(System.in);
```

Comment:
@Hlieb Между операторами всегда ставь пробелы для лучшей читаемости кода!
```suggestion
        Scanner scanner = new Scanner(System.in);
```